mw-webapp/src/component/checkbox/Сheckbox.tsx

```diff
@@ -0,0 +1,37 @@
+import clsx from "clsx";
+import styles from "src/component/checkbox/Сheckbox.module.scss";
+
+/**
+ * Checkbox props
+ */
+interface CheckboxProps {
+
+  /**
+   * Custom class for the checkbox.
+   */
+  className?: string;
+
+  /**
+   * Indicates whether the checkbox is checked.
+   */
+  checked: boolean;
+
+  /**
+   * A callback function to be called when the checkbox's state changes.
+   */
+  onChange: (value?: boolean) => void;
```

Comment:
as I remember we discussed signature of onChange method and agreed on required value parameter, fix please

Reply:
i fix it